src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -1,645 +1,11 @@
-using Microsoft.Build.Framework;
-using Newtonsoft.Json;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System;
-using System.Runtime.InteropServices;
-using System.Text.RegularExpressions;
-using System.Diagnostics;
-using System.Threading;
-
 namespace Microsoft.DotNet.Build.Tasks
 {
     public class CleanupVSTSAgent : BuildTask
     {
-        public bool Clean { get; set; }
-
-        public bool Report { get; set; }
-
-        [Required]
```

Comment:
If there are any targets files calling these properties they will start to fail after this change. If we want to make it less impact-ful you should keep the properties.

Reply:
Added new PR: https://github.com/dotnet/buildtools/pull/1998